autodns.py

```
@@ -102,37 +127,56 @@ def update_dns_web():
-    response = requests.get(f"{CF_API_URL_BASE}?name={dns_record}&type=A", headers=headers)
+    response = requests.get(
+        f"{CF_API_URL_BASE}?name={dns_record}&type=A", headers=headers
+    )
     if response.status_code == 200 and response.json()["result"]:
         dns_record_id = response.json()["result"][0]["id"]
-        update_response = requests.put(f"{CF_API_URL_BASE}/{dns_record_id}", headers=headers, json=data)
+        update_response = requests.put(
+            f"{CF_API_URL_BASE}/{dns_record_id}", headers=headers, json=data
+        )
         if update_response.status_code == 200:
             mapping[guid]["lastUpdated"] = datetime.now().isoformat()
```

Ruff (0.15.4)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)